### PR TITLE
Update global data process

### PR DESCRIPTION
**Pull request by angehung5**
Changes for global_data_process.py:
- Remove wget functions
- Update canopy variable names (fh -> ch, ffrac -> canfrac) to be consistent with new global files
- Add statements about the updates

Put the global data process components back in python README and slurm script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,6 +175,8 @@ The Canopy-App input data in [Table 2](#table-2-canopy-app-required-input-variab
 
 **Global GFS meteorological and canopy files may be provided by request: `Patrick.C.Campbell@noaa.gov`**
 
+**For NOAA Hera users, global canopy files are availabe at `/scratch1/RDARCH/rda-arl-gpu/Wei-ting.Hung/Global_canopy/canopy_app_2022`**
```
Comment (drnimbusrain) [inline README.md]:
```suggestion
**For NOAA Hera users, daily global canopy files for 2022 at 12 UTC are available at `/scratch1/RDARCH/rda-arl-gpu/Wei-ting.Hung/Global_canopy/canopy_app_2022`**
**For GMU Hopper users, daily global canopy files for 2022 at 12 UTC are available at ``**
```
I notice that the global canopy files are not located in previous Hopper location:  /groups/ESS/whung/canopy_wind/gfsv16_test_data/test_2022/canopy*.  Where are these files now staged on Hopper, and can you please add the location above?  I would stage them similarly in what's included and directory structure as you have done for Hera.

Comment (drnimbusrain) [reply README.md]: Also I would move these lines down below my Gridding Machine comments.

File: python/README.md [hunk 2]
```diff
@@ -52,3 +52,86 @@ cases = config_cases(
 ds = run_config_sens(cases)
 ```
 :point_up: We still get a single output dataset, but it has a `case` dimension.
+
+### Generating global data
+
+You can also generate global gridded canopy-app inputs using Python.
```
Comment (drnimbusrain) [inline python/README.md]:
```suggestion
You can also generate and run global gridded canopy-app inputs using Python.  Note:  The python script here assumes that the user already has the global canopy data files (available by request or as staged on NOAA Hera or GMU Hopper) and GFS meteorological files (available by request or on NOAA Hera/HPSS) locally available (under the 'path' working directory set in the python script).  The user may generate their own gridded, meteorological NetCDF files conforming to the format provided here in the GFS examples and README. 
```

File: python/README.md [hunk 3]
```diff
@@ -52,3 +52,86 @@ cases = config_cases(
...
+   nlat        =  1536
+   nlon        =  3072
+   ```
+
+2. Edit namelist (`namelist.canopy`) for correctly named global input files and output prefix
+
+   For example:
+   ```
+   file_vars    = 'input/gfs.t12z.20220630.sfcf023.canopy.nc' 'input/gfs.t12z.20220701.sfcf000.canopy.nc' 'input/gfs.t12z.20220701.sfcf001.canopy.nc'
+   file_out     = 'output/2022-07-01-11-0000_global'
```
Comment (drnimbusrain) [inline python/README.md]:
```suggestion
   file_out     = 'output/global'
```

**Review by drnimbusrain — CHANGES_REQUESTED** [review]
@angehung5 Thank you for modifying and brining this back!  I have some comments and changes.

File: README.md (hunk 1 above)
Comment (angehung5) [reply README.md]: They are at the same location but I changed the filenames to be consistent with the updated python script.

File: README.md [hunk 4]
```diff
@@ -175,6 +175,9 @@ The Canopy-App input data in [Table 2](#table-2-canopy-app-required-input-variab
 
 **Global GFS meteorological and canopy files may be provided by request: `Patrick.C.Campbell@noaa.gov`**
 
+**For NOAA Hera users, daily global canopy files for 2022 at 12 UTC are available at `/scratch1/RDARCH/rda-arl-gpu/Wei-ting.Hung/Global_canopy/canopy_app_2022`**
+**For GMU Hopper users, daily global canopy files for 2022 at 12 UTC are available at `/groups/ESS/whung/canopy_wind/gfsv16_test_data/test_2022`**
+
 Hourly gridded GFSv16 data is available from March 23, 2021 - Current Day and is supplemented by calculated and canopy parameters shown in Table 2.
 
 **GriddingMachine:** GriddingMachine is open source database and software for Earth system modeling at global and regional scales.  Data is easily accessible in consistent formats for ease of downloading/processing.  All available datasets may be found at:  https://github.com/CliMA/GriddingMachine.jl. ([Wang et al., 2022](https://doi.org/10.1038/s41597-022-01346-x)).
```
Comment (drnimbusrain) [inline README.md]:
```suggestion
Hourly gridded GFSv16 data is available from March 23, 2021 - Current Day and is supplemented by calculated and canopy parameters shown in Table 2.

**GriddingMachine:** GriddingMachine is open source database and software for Earth system modeling at global and regional scales.  Data is easily accessible in consistent formats for ease of downloading/processing.  All available datasets may be found at:  https://github.com/CliMA/GriddingMachine.jl. ([Wang et al., 2022](https://doi.org/10.1038/s41597-022-01346-x)).

**For NOAA Hera users, daily global canopy files for 2022 at 12 UTC are available at `/scratch1/RDARCH/rda-arl-gpu/Wei-ting.Hung/Global_canopy/canopy_app_2022`**
**For GMU Hopper users, daily global canopy files for 2022 at 12 UTC are available at `/groups/ESS/whung/canopy_wind/gfsv16_test_data/test_2022`**
```

**Review by drnimbusrain — CHANGES_REQUESTED** [review]
@angehung5 Thank you for the changes.  Just one more small change.

**Review by drnimbusrain — APPROVED** [review]

**Pull request merged**